earth_enterprise/rpms/build.gradle

```diff
@@ -62,15 +90,18 @@ if (new File('build.properties').exists()) {
     }
 }
 
-def openGeeVersionFile = new File("../src/NATIVE-DBG-x86_64/gee_long_version.txt")
+// Create a file handle for gee_long_version_txt
+def openGeeVersionFile = new File( gee_long_version_txt )
 
-if (!openGeeVersionFile.exists()) {
+// Call scons to create the file 'gee_long_version.txt'
+// The actual parameter used by scons can remain 'release=1'
+if ( !openGeeVersionFile.exists() ) {
     GeeCommandLine.expand(
-            ["/usr/bin/scons", "release=1", "NATIVE-DBG-x86_64/gee_long_version.txt"],
-            "Runing `scons` to produce <gee_long_version.txt> failed!",
-            null, // Use default environment variables.
-            new File("../src") // Working directory.
-        )
+      ["/usr/bin/scons", "release=1", gee_long_version_txt ],
```

Comment:
This command only works for a `gee_long_version.txt` path under the `NATIVE-REL-x86-64` build directory, and will fail for all other options.

You need to pass `debug`, `release`, `optimize` and `profile` back to `scons` to produce the output.

Here's how to test you code, e.g. for building with `debug=1`:

* Remove your `src/NATIVE-DBG-x86_64/` directory (as well as the other build directories like `src/NATIVE-REL-x86_64` for the purposes of testing).
* `cd earth_enterprise/rpms`
* `./gradlew -Pdebug tasks`
* If you see the following error, this code doesn't work correctly.

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/pav/prod/bitbucket.com/t-sciences/geoplatform/earthenterprise_code/earthenterprise/earth_enterprise/rpms/build.gradle' line: 103

* What went wrong:
A problem occurred evaluating root project 'rpms'.
>                 Runing `scons` to produce <../src/NATIVE-DBG-x86_64/gee_long_version.txt> failed!
                  Command: [/usr/bin/scons, release=1, ../src/NATIVE-DBG-x86_64/gee_long_version.txt]
                  Exit code: 2.
                  Error output:
                  
  scons: warning: The build_dir keyword has been deprecated; use the variant_dir keyword instead.
  File "/home/pav/prod/bitbucket.com/t-sciences/geoplatform/earthenterprise_code/earthenterprise/earth_enterprise/src/SConstruct", line 519, in <module>
  scons: *** Do not know how to make File target `NATIVE-DBG-x86_64/gee_long_version.txt' (/home/pav/prod/bitbucket.com/t-sciences/geoplatform/earthenterprise_code/earthenterprise/earth_enterprise/src/NATIVE-DBG-x86_64/gee_long_version.txt).  Stop.
  
                  Standard output:
                  scons: Reading SConscript files ...
  Checking for C library gtest... (cached) no
  Checking for C header file libpng12/png.h... (cached) yes
  scons: done reading SConscript files.
  scons: Building targets ...
  scons: building terminated because of errors.
```